packages/react-link/src/common/isConformant.ts

```diff
@@ -0,0 +1,17 @@
+import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
+
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
+  const defaultOptions = {
+    disabledTests: [
+      `has-docblock`,
+      `kebab-aria-attributes`,
+      `as-renders-fc`,
+      `as-renders-react-class`,
+      `as-passes-as-value`,
```

Comment:
i dont think we should have these disabled.

Reply:
There seems to be a function component in link that is not using forwardRef. It could be possible that the old compose link is using is responsible, but I will dig in more. Here is the error isConformant runs into after link runs the as tests:
```  jsx
    [console.error] Error: Uncaught [Error: [console.error] Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?%s%s

    Check the render method of `LinkBase`.
        in MyComponent (created by LinkBase)
        in LinkBase (created by StyledLinkBase)
        in StyledLinkBase (created by WrapperComponent)
        in WrapperComponent]

      19 |     throw args[0];
      20 |   } else {
    > 21 |     throw new Error(`[console.${type}] ${args.join(' ')}`);
         |           ^
      22 |   }
      23 | }
      24 |
```

Reply:
I figured out what was causing the remaining errors and the 'as' tests are now enabled in `link`.